requirements.txt

```diff
@@ -6,4 +6,5 @@ pandas==0.25.3
 pykalman==0.9.5
 scipy==1.2.3
 statsmodels==0.10.2
-scikit-learn==0.19.2
+scikit-learn==0.24.2
+decorator==5.1.0
```

Comment:
Why do we need to add decorator? Do we know where we need it?

Reply:
bayesian_changepoint/offline_changepoint_detection needs it and our test were failing because of that.